components.json

```diff
@@ -1185,6 +1185,11 @@
 			"description": "Line number at the beginning of code lines.",
 			"owner": "kuba-kubula"
 		},
+		"lines": {
+			"title": "Lines",
+			"description": "Introduces HTML markup around physical lines to allow styling of the particular line.",
```

Comment:
This seems more complicated than it has to be. How about: "Adds wrappers around lines to allow line-bases styling"?